### PR TITLE
add an AUTHORS file

### PR DESCRIPTION
**Pull request by devoncarew**
@ggirou

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Below is a list of people and organizations that have contributed
+# to the Stagehand project. Names should be added to the list like so:
```
Comment (lukechurch) [inline AUTHORS]: s/Stagehand/dart-pad ?

Comment (devoncarew) [inline AUTHORS]: Opps, done.

**Comment by ggirou** [comment]
Use this email instead: guillaume.girou@gmail.com

**Comment by ggirou** [comment]
Thanks :)

**Pull request merged**